test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -107,337 +107,337 @@ def _test_auto_tracing(
             traced_rewritten_out = traced_rewritten(*example_args)
             self.assertTrue(_allclose(traced_rewritten_out, out_q))
 
-
 @skipIfNoFBGEMM
-class TestQuantizeDBR(QuantizeDBRTestCase):
-    def test_fusion(self):
+class TestQuantizeDBRIndividualOps(QuantizeDBRTestCase):
+    """
+    Tests that DBR quantization covers individual ops
+    """
+    def test_conv(self):
```

Comment:
do we want to reuse/merge these test cases with other tools? like fx graph mode quant?

Reply:
I feel like that would be a good task to do when we approach OSS beta, might be too early now